[State Mgmt] Docset refresh (#2541)
dapr/docs: daprdocs/content/en/developing-applications/building-blocks/state-management/howto-state-query-api.md

  }
}
```

To compare the value of the color code, the key will be `shape.color.code`. 

If the `filter` section is omitted, the query returns all entries.

### `sort`

The `sort` is an ordered array of `key:order` pairs, where:

- `key` is a key in the state store
- `order` is an optional string indicating sorting order:
  - `"ASC"` for ascending
  - `"DESC"` for descending  
  If omitted, ascending order is the default.

### `page`

The `page` contains `limit` and `token` parameters.

- `limit` sets the page size.
- `token` is an iteration token returned by the component, used in subsequent queries.

Behind the scenes, this query request is translated into the native query language and executed by the state store component.

## Example data and query

Let's look at some real examples, ranging from simple to complex.

As a dataset, consider a [collection of employee records](../query-api-examples/dataset.json) containing employee ID, organization, state, and city. Notice that this dataset is an array of key/value pairs, where:

- `key` is the unique ID
- `value` is the JSON object with employee record.

> To better illustrate functionality, organization name (org) and employee ID (id) will be a nested JSON person object.

Reviewer: ```suggestion
To better illustrate the functionality, organization name (org) and employee ID (id) are a nested JSON person object.
```
